pyproject.toml

```diff
@@ -28,17 +28,20 @@ rich = "^13.6.0"
 scikit-image = "^0.22.0"
 "ruamel.yaml" = "^0.18.6"
 plotly = "^5.16.1"
+pandas = "^2.1.0"
 
 [tool.poetry.dev-dependencies]
 pytest = ">=8.1.1"
 coverage = ">=7.0.1"
 pytest-mock = "^3.6.0"
-pandas = "^2.1.0"
 joblib = "^1.3.2"
 future = "^0.18.3"
 flake8 = ">=4.0.1"
 auxiliary = ">=0.1.0"
 tqdm = ">=4.62.3"
+SimpleITK = "^2.2.2"
+torch = "^2.1.0"
+nibabel = "^5.1.0"
```

Comment:
This is redundant, as all functionality related to nibabel is taken care of by SimpleITK.

```suggestion
```